backend/go-server/go.mod

```diff
@@ -0,0 +1,33 @@
+module main
```

Comment:
usually golang modules should be named with their version control repo name (i.e. `module github.com/PUArallelepiped/PUN-street-Universal-Access`)

Reply:
k, you're right